### PR TITLE
Add French translation of nodes

### PR DESCRIPTION
**Pull request by GogoVega**
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)

## Proposed changes

Add French translation of nodes.
Follows [PR#3962](https://github.com/node-red/node-red/pull/3962) (editor's translation).

## Checklist

- [x] I have read the [contribution guidelines](https://github.com/node-red/node-red/blob/master/CONTRIBUTING.md)
- [ ] For non-bugfix PRs, I have discussed this change on the forum/slack team.
- [x] I have run `grunt` to verify the unit tests pass
- [ ] I have added suitable unit tests to cover the new/changed functionality

### CODE REVIEW

**Comment by Steve-Mcl** [comment]
@GogoVega do you know any French speaking users in the community who can review your works?

Unfortunately we (I) are not fluent in french and cannot verify your translations.

V3.1 release is not too far off so the sooner we can get a French speaking reviewer the more chance your translations will be included in the release.

Thank you very much for your contributions.

File: packages/node_modules/@node-red/nodes/locales/fr/common/60-link.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <p>Le noeud peut être connecté à n'importe quel noeud <code>Lien sortant</code> qui existe sur n'importe quel onglet.
+      Une fois connectés, ils se comportent comme s'ils étaient câblés ensemble.</p>
+    <p>Les fils entre les noeuds de lien ne s'affichent que lorsqu'un noeud de lien est sélectionné.
+      S'il y a des fils vers d'autres onglets, un noeud virtuel est affiché et permet si on clique dessus 
+      d'accéder à l'onglet approprié.</p>
+    <p><b>Remarque</b> : Impossible de créer des liens entrant ou sortant d'un sous-flux.</p>
+</script>
+
+<script type="text/html" data-help-name="link out">
+    <p>Créer des liens virtuelles entre les flux.</p>
```
Comment (golfvert) [inline packages/node_modules/@node-red/nodes/locales/fr/common/60-link.html:29]: <p>Créer des liens virtuels entre les flux.</p>

File: packages/node_modules/@node-red/nodes/locales/fr/common/90-comment.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<script type="text/html" data-help-name="comment">
+    <p>Un noeud d'annotations, que vous pouvez utiliser pour ajouter des commentaires à vos flux.</p>
+    <h3>Détails</h3>
+    <p>Le panneau d'édition acceptera la syntaxe Markdown. Le texte sera rendu dans
```
Comment (golfvert) [inline packages/node_modules/@node-red/nodes/locales/fr/common/90-comment.html:20]: <p>Le panneau d'édition accepte la syntaxe Markdown.

File: packages/node_modules/@node-red/nodes/locales/fr/common/60-link.html (hunk 1 above)
Comment (golfvert) [inline packages/node_modules/@node-red/nodes/locales/fr/common/60-link.html:29]: <p>Créer des liens virtuels entre les flux.</p>

**Review by golfvert — CHANGES_REQUESTED** [review]
Just two minot editorial changes. Overall very good translation.

**Comment by GogoVega** [comment]
@golfvert, thank you for your message and your work 🙂.

**Comment by GogoVega** [comment]
Should I move the target branch from `master` to `dev`?

**Comment by knolleary** [comment]
Hi - master is fine. Thank you all for the work on this. We'll get it merged in time for the 3.1-beta release

**Pull request merged**